### DISCUSSION TITLE
if chia is like Bitcoin UTXO model,the Satoshi is the smallest unit of Bitcoin,what is the smallest unit chia?

### DISCUSSION THREAD

**Discussion by shendejun** [Chia-Network/chia-blockchain#5910] [Support]
if chia is like Bitcoin UTXO model,the Satoshi is the smallest unit of Bitcoin,what is the smallest unit chia?

**Comment by tiny0little** [comment]
mojo

**Comment by BrandtH22** [comment]
@shendejun , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
